dist/utils/commas.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 function numberWithCommas(x) {
-  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  return x && x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
Is this really necessary?

Reply:
yes. when x is undefined, this returns an error. You can try it with a package that doesn't return `github` data.